### DISCUSSION TITLE
CustomRestControllerAdvice 커스텀 애노테이션 변수명 짓기

### DISCUSSION THREAD

**Discussion by june-777** [woowa-techcamp-2024/Team1-3K1K#27] [Polls]
### 개요
GlobalExceptionHandler에서 모든 예외를 핸들링하면, 예외의 Context를 파악하기 어려운 트레이드오프가 있다고 결론내려졌습니다.
따라서, `RestControllerAdvice`를 활용해, 각각의 컨트롤러에서 예외를 핸들링하도록 논의되었습니다.

@Order 를 공통적으로 명시해야하는 문제가 있어서, RestControllerAdvice 를 래핑한 커스텀 애노테이션을 만들기로 하였고, 그에 대한 변수명 투표입니다!

**Comment by kimhyun5u** [comment]
제가 멋대로 지었습니다.✨`DomainExceptionHandler`✨

**Reply by june-777** [reply]
DomainExceptionHandler 네이밍도 괜찮네요! 👍
